dev/com.ibm.websphere.appserver.features/visibility/public/mpHealth-4.0/com.ibm.websphere.appserver.mpHealth-4.0.feature

```diff
@@ -0,0 +1,25 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.mpHealth-4.0
+visibility=public
+singleton=true
+IBM-App-ForceRestart: install, \
+ uninstall
+IBM-API-Package: \
+  org.eclipse.microprofile.health;  type="stable", \
+  org.eclipse.microprofile.health.spi;  type="stable"
+IBM-ShortName: mpHealth-4.0
+Subsystem-Name: MicroProfile Health 4.0
+-features=io.openliberty.org.eclipse.microprofile.health-4.0,\
+ io.openliberty.cdi-3.0,\
+ com.ibm.websphere.appserver.jndi-1.0,\
+ com.ibm.websphere.appserver.json-1.0,\
+ com.ibm.websphere.appserver.servlet-5.0,\
+ com.ibm.websphere.appserver.mpConfig-3.0,\
+ com.ibm.wsspi.appserver.webBundle-1.0,\
+ io.openliberty.mpCompatible-5.0
+-bundles=com.ibm.websphere.jsonsupport, \
+ io.openliberty.microprofile.health.3.1.internal.jakarta; apiJar=false; location:="lib/", \
```

Comment:
Is this a new bundle?  Can't find it.

Reply:
When we add the Jakarta transformer property `jakartaeeMe: true` to the bnd.bnd file, when we build the bundle, it would create an additional bundle jar with `jakarta` appended to it with the javax->jakarta package names renamed.